tests/helpers/load_difficulty_tests.py

```diff
@@ -1,19 +1,9 @@
-from __future__ import annotations
-
 import json
 import os
 from importlib import import_module
 from typing import Any, Dict, List
 
-from ethereum.base_types import U256, Uint
-from ethereum.utils.hexadecimal import (
-    hex_to_bytes,
-    hex_to_bytes8,
-    hex_to_bytes32,
-    hex_to_hash,
-    hex_to_u256,
-    hex_to_uint,
-)
+from ethereum.utils.hexadecimal import hex_to_u256, hex_to_uint
```

Comment:
Interesting that `flake8` did not catch this. Will create an issue to find more such cases